lib/rtcpeerconnection.js

```diff
@@ -422,12 +429,22 @@ RTCPeerConnection.prototype.close = function close() {
 };
 
 /**
- * Construct an {@link RTCSessionDescription} containing an SDP answer.
- * @param {?function} [onSuccess]
- * @param {?function} [onFailure]
+ * onSuccess callback for {@link RTCPeerConnection.createAnswer}
+ * @callback onCreateAnswerSuccess
+ * @param {RTCSessionDescription}
+ *//**
```

Comment:
I think it'd be OK to just refer to the [RTCSessionDescriptionCallback](http://w3c.github.io/webrtc-pc/#dom-rtcsessiondescriptioncallback) type rather than introduce the `@callback` here.